style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background: #191765;
+  color: white;
+  font-family: "Roboto", sans-serif;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+/*
+  Header
+*/
```

Comment:
- [ ] Just as mentioned above, It would be nice if we keep our code base clean by removing commented codes